Reviews/337 -  Royal web wallet and casper connector/1- Development/README.md

```diff
@@ -0,0 +1,394 @@
+································|·············|·············|·············|···············|··············
+|  YourContract                 ·          -  ·          -  ·    1074922  ·        3.6 %  ·          -  │
+·-------------------------------|-------------|-------------|-------------|---------------|-------------·
+
+  2 passing (4s)
+
+Done in 15.29s.
+
+```
+
+## Unit / Automated Testing
+
+The project contains only 2 unit tests. These tests run successfully. There are no negative path tests at all.
+
+Requirement | Finding
+------------ | -------------
+Unit Tests - At least one positive path test | Pass with notes
+Unit Tests - At least one negative path test | Failed
+Unit Tests - Additional path tests | Failed
+
+# Documentation
+
+### Code Documentation
+
+The code is overall well documented, but the reviewer thinks that some improvements are needed  :
+
+**- Unused code can be removed in order to improve code readability (eg : hardhat/contracts/00_deploy_your_contract.js).** 
+
+**- Secret informations (like API KEY) should not be committed to github (some "api key" found hardcoded in the repository (eg : Alchemy))**
+  
+Requirement | Finding
+------------ | -------------
+Code Documented | PASS with note
+
+### Project Documentation
+
+ There is a lack of documentation at several levels :
+- There is no detailed documentation describing the project, its purpose and its main features
+- It's hard to understand and to use the "configure variables" part in the README file
+- There is no documentation on unit tests and  how to run them
+- There is no documentation on the errors that can occur while running and testing the application and how to troubleshoot them.
+- There is  no step-by-step usage documentation on the webapp : how it works, what is a Supervisor, etc.
+
+Requirement | Finding
+------------ | -------------
+Usage Documented | Failed
+Example Documented | Failed
+
+
+## Overall Conclusion on Documentation
+
+In the reviewer's opinion, the project and usage documentation are not sufficient. 
+
+# Open Source Practices
+
+## Licenses
+
+In the grant, it's stated  that the license is an  "Apache License 2.0" one.
+But the repository indicates usage of  a "MIT License" one.
+
+Requirement | Finding
+------------ | -------------
+OSI-approved open source software license | Failed
+
+## Contribution Policies
+
+No Contributing and Security policy
+
+Requirement | Finding
+------------ | -------------
+OSS contribution best practices | Failed 
+
+
+# Coding Standards
+
+## General Observations
+
+The project builds and the webapp runs successfully. **However the lack of documentation makes it impossible to  do an "exchange" and to run a complete test case**.
+
+# Final Conclusion
+
+**The project needs serious improvements on  documentation.
+The licence  informations provided in the repository are wrong. There are no Contributing  or Security policies attached to the repository.
```

Comment:
By looking at the review, it seems like there are more reasons for the failed review, like failure to meet the acceptance criteria. I would be good to also mention them here.